runner/src/test/resources/application.properties

```diff
@@ -6,6 +6,4 @@ kafka.security.protocol=PLAINTEXT
 kafka.sasl.jaas.config=none
 
 # SSO
```

Comment:
Can we set environment variable placeholders here that can be set on the `Deployment` resource definition e.g:

```
quarkus.oidc.auth-server-url=${EVENT_BRIDGE_AUTH_SERVER_URL:some-default}
```

Reply:
@robpblake it is already like this. This is the `application.properties` of the test package

Reply:
See https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/manager/src/main/resources/application.properties

Reply:
@r00ta I don't think I should review PRs in GitHub IDE any more 😆